### PR TITLE
Implement Sign In Screen UI

### LINKED ISSUE #7: Implement Sign In Screen UI
The Sign in screen UI is ready

Image

### PR DESCRIPTION
**Pull request by truongtdph40209**
## Related issues
Resolved #7 

## Summary
Implement Sign In Screen UI

## How to test
Run app

## Screenshots
image

### CODE REVIEW

File: components/molecules/BackCustom.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+}
+const BackCustom: React.FC<BackCustomProps> = ({ onPress }) => {
+  const colors = getColors(useColorScheme())
+
+  return (
+    <YStack
+      marginTop={60}
+    >
+      <XStack>
+        <TouchableOpacity
```
Comment (NguyenVanViet2004) [inline components/molecules/BackCustom.tsx:19]: Use button of tamagui library

File: components/molecules/ButtonCustom.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import { Text, XStack, YStack } from 'tamagui'
+
+interface ButtonCustomProps {
+  onPress?: () => void
+  buttonText?: string
+  textColor?: string
+  backgroundColor?: string
+  icon?: JSX.Element
+}
+const ButtonCustom: React.FC<ButtonCustomProps> = (
```
Comment (NguyenVanViet2004) [inline components/molecules/ButtonCustom.tsx:12]: Replace this component with [positive button](https://github.com/NguyenVanViet2004/FE-Shoes-Ecommerce/commit/9e27cc47bd61e01573579980e6a7807a5ec796f9#diff-ef53c3b28f7c308ae80d0911fb094b34a74aef50c7e89647df588a04ff1b48fa)

File: components/molecules/InputCustom.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      <XStack
+      >
+        <Input
+          {...props}
+          paddingHorizontal={16}
+          borderRadius={50}
+          height={48}
+          backgroundColor={'#FFF'}
+          alignItems="center"
+          width={'100%'}
```
Comment (NguyenVanViet2004) [inline components/molecules/InputCustom.tsx:35]: width='100%'

File: components/molecules/InputCustom.tsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+          borderWidth={0}
```
Comment (NguyenVanViet2004) [inline components/molecules/InputCustom.tsx:36]: borderWidth=0

File: components/molecules/InputCustom.tsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+          color={'black'}
```
Comment (NguyenVanViet2004) [inline components/molecules/InputCustom.tsx:37]: color ?

File: components/templates/SignInTemplate.tsx [hunk 6]
```diff
@@ -0,0 +1,166 @@
+import { Feather } from '@expo/vector-icons'
+import React, { useEffect, useState } from 'react'
+import { SafeAreaView, StyleSheet, TouchableOpacity, useColorScheme, View } from 'react-native'
+import { Image, Text } from 'tamagui'
+
+import getColors from '~/constants/Colors'
+
+import BackCustom from '../molecules/BackCustom'
```
Comment (NguyenVanViet2004) [inline components/templates/SignInTemplate.tsx:8]: Use relative path

File: components/templates/SignInTemplate.tsx [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+import useTranslation from './../../hooks/useTranslation'
+
+const SignInTemplate: React.FC = (): JSX.Element => {
+  const [showPass, setShowPass] = useState<boolean>(true)
+  const [email, setEmail] = useState<string>('')
+  const [password, setPassword] = useState<string>('')
+  const { t } = useTranslation()
+  const colors = getColors(useColorScheme())
+  useEffect((): void => {
+    console.log(email)
```
Comment (NguyenVanViet2004) [inline components/templates/SignInTemplate.tsx:20]: Why do we need console.log here? Aim to do?

File: components/templates/SignInTemplate.tsx [hunk 8]
```diff
@@ -0,0 +1,166 @@
...
+    console.log(password)
+  }, [email, password])
+
+  const togglePass = (): void => {
+    setShowPass(!showPass)
+  }
+
+  const handleBack = (): void => {
+    console.log('Click to back')
```
Comment (NguyenVanViet2004) [inline components/templates/SignInTemplate.tsx:29]: ??????

File: components/templates/SignInTemplate.tsx [hunk 9]
```diff
@@ -0,0 +1,166 @@
...
+  }
+
+  return (
+    <SafeAreaView
+      style={[styles.container, { backgroundColor: colors.lightSilver }]}
+    >
+      <BackCustom onPress={handleBack} />
+      <View style={styles.header}>
+        <Text fontSize={28} fontWeight="500" color={colors.midnightBlue}>
+          {t('Hello Again')}
```
Comment (NguyenVanViet2004) [inline components/templates/SignInTemplate.tsx:47]: en.json and vi.json ?????

File: components/molecules/InputCustom.tsx (hunk 4 above)
Comment (truongtdph40209) [reply components/molecules/InputCustom.tsx:36]: Default borderWidth is 1
image

**Pull request closed without merging**